magento/pwa-studio
# [Doc] fix reference doc generator

## Description

This PR fixes a bug in the reference doc generator caused by the migration of the Page Builder components into a separate package.

## Related Issue

Closes PWA-391

## Acceptance 

@bdenham 

### Verification Stakeholders

@bdenham 

### Verification Steps

1. navigate to the devdocs project
1. Clean the project: `npm run clean`
1. Build reference docs: `npm run build:reference-doc-snippets`
1. Verify reference docs are generated with no errors under `src/_includes/auto-generated`
1. Start the preview server: `npm run start`
1. Verify project builds and starts with no errors

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
